docker/Dockerfile.test

```diff
@@ -13,14 +13,16 @@ RUN apt-get update && apt-get install -y ca-certificates gnupg software-properti
 # Install Python 3.9 and python3-pip
 RUN apt-get install -y python3.9 python3.9-venv python3.9-dev python3-pip
 
-ADD requirements.txt .
-ADD requirements.dev.txt .
+ADD pyproject.toml .
+ADD README.md .
+ADD tails_server ./tails_server
 
+RUN pip3 install --upgrade pip
 # Upgrade pip using Python 3.9 to ensure compatibility
 RUN python3.9 -m pip install --upgrade pip
```

Comment:
We should probably use the same version of python used in the base image to test things. We were stuck with using Ubuntu 20.04 and an older version of python due to the dependency on libindy, we should now be able to change this as well.

Reply:
Disregard the comment, I got things mixed-up: this is for a follow-up tasks, when we drop `indy-sdk` as a test dependency.